ports/servoshell/webview.rs

```diff
@@ -564,6 +600,31 @@ where
                 EmbedderMsg::WebViewBlurred => {
                     self.focused_webview_id = None;
                 },
+                EmbedderMsg::WebViewPaintingOrder(webview_ids) => {
+                    self.painting_order = webview_ids;
+
+                    if let Some(&newest_webview_id) = self.creation_order.last() {
+                        let mut newest_webview_is_visible = false;
+
+                        // Hide any visible browsers other than the most recently created.
+                        // TODO stop doing this once we have full multiple browser support
```

Comment:
```suggestion
                        // TODO: Stop doing this once we have full multiple browser support.
```

Reply:
Fixed in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)